### PR TITLE
Assures that an email address is accessible to the send method.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Note that the goal of this PR is to do the minimal amount of work to fix the issue without possibly breaking any other behavior.

TODO:
- Refactor `bin/cron/deliver_poste_messages`.
- Write thorough tests.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Need to confirm this does what I want without breaking anything. That said, feel welcome to look at this before I convince myself of this. It would be really good to get this in by the AM DTT and important to get this in by the PM DTT.

File: bin/cron/deliver_poste_messages [hunk 1]
```diff
@@ -127,7 +127,9 @@ class Deliverer
 
     message = StringIO.new
 
-    to_address = parse_address(header['to'], recipient)
+    to_address = parse_address(header['to'], recipient.merge({
+      poste_deliveries_contact_email: recipient[:contact_email]
```
Comment (aoby) [inline bin/cron/deliver_poste_messages]: Isn't this just copying the value from `:contact_email` to `:poste_deliveries_contact_email`? Why is this necessary? Can't we just reference `defaults[:contact_email']` below?

Comment (ashercodeorg) [inline bin/cron/deliver_poste_messages]: Done.

**Comment by ashercodeorg** [comment]
PT(A)L: ready for review.

**Comment by ashercodeorg** [comment]
Though not part of this PR, I'm wondering if we should always be using `delivery[:contact_email]`.

**Comment by aoby** [comment]
LGTM.

RE: always using `delivery[:contact_email]`: yeah, I'm all for simplifying further & consolidating code paths, of course as long as existing functionality remains intact.
